server/monitor/src/main/java/org/apache/accumulo/monitor/Monitor.java

```diff
@@ -589,6 +624,33 @@ private void fetchScans() throws Exception {
     }
   }
 
+  private void fetchCompactions() throws Exception {
+    ServerContext context = getContext();
+    for (String server : context.instanceOperations().getTabletServers()) {
+      final HostAndPort parsedServer = HostAndPort.fromString(server);
+      Client tserver = ThriftUtil.getTServerClient(parsedServer, context);
+      try {
+        var compacts = tserver.getActiveCompactions(null, context.rpcCreds());
+        synchronized (allCompactions) {
+          allCompactions.put(parsedServer, new CompactionStats(compacts));
+        }
+      } catch (Exception ex) {
+        log.debug("Failed to get active compactions from {}", server, ex);
+      } finally {
+        ThriftUtil.returnClient(tserver);
+      }
+    }
+    // Age off old compaction information
+    var entryIter = allCompactions.entrySet().iterator();
+    long now = System.currentTimeMillis();
+    while (entryIter.hasNext()) {
+      var entry = entryIter.next();
+      if (now - entry.getValue().fetched > 5 * 60 * 1000) {
```

Comment:
This should use nanoTime to compute time durations, to avoid system clock adjustments affecting the computation.